examples/repeatergrid/repeatergrid.jl

```diff
@@ -0,0 +1,109 @@
+using QuantumSavory
```

Comment:
add this file to the example test runner, so that we can make sure that it does not break in the future. All other examples are executed by the test runner